tests/unit/SpecialResolution.spec.ts

```diff
@@ -0,0 +1,367 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import VueRouter from 'vue-router'
+import flushPromises from 'flush-promises'
+import sinon from 'sinon'
+import { getVuexStore } from '@/store/'
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import { axios } from '@/utils/'
+import SpecialResolution from '@/views/SpecialResolution.vue'
+import mockRouter from './MockRouter'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore()
+
+// Prevent the warning "[Vuetify] Unable to locate target [data-app]"
+document.body.setAttribute('data-app', 'true')
+
+describe('SpecialResolution component', () => {
+  let wrapper: any
+  const { assign } = window.location
+
+  // Define Session
+  sessionStorage.setItem('PAY_API_URL', 'https://pay.api.url/')
+  sessionStorage.setItem('AUTH_API_URL', 'https://auth.api.url/')
+  sessionStorage.setItem('AUTH_WEB_URL', 'https://auth.web.url/')
+  sessionStorage.setItem('DASHBOARD_URL', 'https://dashboard.url/')
+  sessionStorage.setItem('KEYCLOAK_TOKEN', 'eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJUbWdtZUk0MnVsdUZ0N3' +
```

Comment:
You may not need the KC token for your tests. Usually it's App.vue that needs it. Check and clean up if not needed.

Reply:
without KC token , the test is not passing

Reply:
OK. Maybe it's looking at the "is staff" or "is premium" getters, which I think look at the role from the token.

In your opinion, would it be cleaner to mock the getters to return True according to your test needs, or is it OK to leave the sample KC token here?

Reply:
Looks like its checking only if the value exists, when I am giving any value, its working, not on blank, I will give a sample value (not token) (in all other filing we are using the same token) , if you are good